src/Symfony/Component/Console/Tests/Command/InvokableCommandTest.php

```diff
@@ -39,6 +41,7 @@ public function testCommandInputArgumentDefinition()
             #[Argument] ?string $firstName,
             #[Argument] string $lastName = '',
             #[Argument(description: 'Short argument description')] string $bio = '',
+            #[Argument(description: [self::class, 'generateEducationDescription'])] string $education = '',
```

Comment:
Your example could be written with concatenation of strings instead of `sprintf`, but I guess you oversimplified the example and want to add more logic?
```php
    #[Argument(description: 'Mention your favourite lectures from' . self::UNIVERSITY . '!')]
    string $education,
```
```suggestion
            #[Argument(description: 'Mention your favourite lectures from' . self::UNIVERSITY . '!')]
            string $education = '',
```

Reply:
The point is having a callable – so the inline concatenation does not show the "purpose" of passing a callable.

In my particular situation i'm actually `array_map`ping and `join`–ing `Enum::cases()` to present them in the description, btw.

As for sprintf vs concatenation vs interpolation – sprintf is just my personal preference for readability -> I can change it anyway needed.